Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -135,25 +135,32 @@ struct SurfaceData {
   /// Surface identifier. Not available in the TrackML datasets.
   uint64_t geometry_id;
   /// Partially decoded surface identifier components.
-  uint32_t volume_id, layer_id, module_id;
+  uint32_t volume_id, boundary_id, layer_id, module_id;
   /// Center position components in mm.
   float cx, cy, cz;
   /// Rotation matrix components.
   float rot_xu, rot_xv, rot_xw;
   float rot_yu, rot_yv, rot_yw;
   float rot_zu, rot_zv, rot_zw;
-  /// Limits and pitches in mm. Not always available.
-  float module_t = -1;
-  float module_minhu = -1;
-  float module_maxhu = -1;
-  float module_hv = -1;
-  float pitch_u = -1;
-  float pitch_v = -1;
-
-  DFE_NAMEDTUPLE(SurfaceData, geometry_id, volume_id, layer_id, module_id, cx,
-                 cy, cz, rot_xu, rot_xv, rot_xw, rot_yu, rot_yv, rot_yw, rot_zu,
-                 rot_zv, rot_zw, module_t, module_minhu, module_maxhu,
-                 module_hv, pitch_u, pitch_v);
+
+  // In case it describes a boundary surface
+  int volume_link_o = -1;
+  int volume_link_a = -1;
```

Comment:
What does these variables describe?

Reply:
Can be removed for now, this would be the boundary links if they were unique.